gateway-provider-security-authz-composite/src/main/java/org/apache/knox/gateway/deploy/impl/CompositeAuthzDeploymentContributor.java

```diff
@@ -58,7 +59,7 @@ public void contributeFilter( DeploymentContext context, Provider provider, Serv
 
     Map<String, String> providerParams = provider.getParams();
     String providerNames = providerParams.get("composite.provider.names");
-    if (!providerNames.isEmpty()) {
+    if (!StringUtils.isEmpty(providerNames)) {
```

Comment:
Good job and thank you for the changes @harshiljhaveri. One last thing is that this conditional seems to be unnecessary. The `parseProviderNames` returns an empty list if the parameter is blank so the for loop won't do anything in that case. I would just remove this `if` statement and rely on that.

Reply:
Nice catch Attila! I missed it previously. Thanks for spotting this out!